### ISSUE TITLE
include path problem

### ISSUE THREAD

**Issue by Muzza** [slynch8/10x#945]
Autocomplete has largely stopped working for me in projects where it did used to work, and I've narrowed it down to include paths.

Take these three files using the full paths:

```cpp
// FILE: Project/common/LibB/LibB.h
#define PROP( x ) int x;
```

```cpp
// FILE: Project/common/LibA/LibA.h
#include "../LibB/LibB.h" // causes problems
// #include "../common/LibB/LibB.h"	// fixes the problem
```

```cpp
// FILE: Project/App/App.cpp
#include "../common/LibA/LibA.h"

struct S
{
	PROP( x ) // not correctly parsed UNLESS LibA.h changes the include path
	int a;
};

void Test( void )
{
	S s;
	// s.x not found by autocomplete UNLESS LibA.h changes the include path
}
```
App.cpp includes LibA.h
LibA.h includes LibB.h using a path relative to itself, but not relative to App.cpp. The parser does not seem to find this file.
If you change LibA.h to include LibB.h using a path relative to App.h, then it all starts working.

Note, that whenever you change the include paths, you need to delete the parser cache and restart 10x for the change to be fully reflected. (which is I presume, another bug)

**Comment by slynch8** [comment]
Taking a look at this now. I'm trying to create a simple test project but I can't get it to break. Are the header files included in the project? And are you using a sln or 10x workspace?

**Comment by slynch8** [comment]
Ignore that, I've got it now

**Comment by slynch8** [comment]
Have you added the Common folder path to your additional include paths in the project? Otherwise I don't see how the compiler would find it either. If so then 10x should be picking that up, so that may be the bug.

**Comment by slynch8** [comment]
Ignore that last comment as well. It seems that the compiler does check the path relative to the current file in the chain, I didn't know that.

**Comment by slynch8** [comment]
fixed (0.32.80)
I couldn't repro the includes not updating until restart and cache delete. If this is still happening let me know and I'll take another look.
